mypy/build.py

```diff
@@ -294,7 +298,9 @@ def default_lib_path(data_dir: str,
     # Add fallback path that can be used if we have a broken installation.
     if sys.platform != 'win32':
         path.append('/usr/local/lib/mypy')
-
+    assert path, "Could not resolve typeshed subdirectories. If you are using MyPy " \
```

Comment:
Is there a way to turn this into a regular (blocking) error message? Many people don't read the error when they get a traceback, and just instantly file a bug or go to a help forum.